### DISCUSSION TITLE
Multiple instances of the same Livewire component as wizard steps

### DISCUSSION THREAD

**Discussion by GREPY** [spatie/laravel-livewire-wizard#117] [Q&A]
Hello, my question is quite simple - can I use multiple instances of the same Livewire components as wizard steps?

I'm creating a quiz, each quiz will be a series of tasks / questions, and I thought of using this library for those. Problem is that many of the tasks / questions shares the same Livewire components - for example test contains one task of certain type (component `TaskTypeA`) that another of different type (component `TaskTypeB`) but the third step is again `TaskTypeA`, only with different data loaded.

Is something like this possible using this library or should I look somewhere else?

**Comment by ju5t** [comment]
Did you have any issues with it?

If you do, you could just extend `TaskTypeA` into a new class and add that to the step so you're not duplicating any code.
